website/screens/utilities/halstackProvider/HalstackProviderPage.tsx

```diff
@@ -0,0 +1,175 @@
+          <Link href="/theme-generator/opinionated-theme" passHref>
+            <DxcLink>opinionated theme generator</DxcLink>
+          </Link>{" "}
+          to create the theme object that will be used in the{" "}
+          <Code>HalstackProvider</Code>.
+        </DxcParagraph>
+        <DxcParagraph>
+          Here is an example of customizing the colours of a DxcAccordion and a
+          DxcTextInput:
+        </DxcParagraph>
+        <Example example={opinionatedTheme} defaultIsVisible />
+
+        <DxcParagraph>
+          As you can see in the example above, we create the{" "}
+          <Code>customTheme</Code> object with as many components as we want to
+          customize and the values we want to customize. Then we pass this
+          object to the <Code>HalstackProvider</Code>, which wraps our
+          components, through its <Code>theme</Code> property.
+        </DxcParagraph>
+      </>
+    ),
+  },
+  {
+    title: "Advanced theme",
+    content: (
+      <>
+        <DxcParagraph>
+          If we need a higher level of customisation you need to choose to use
+          the Advanced Theme as explained in the{" "}
+          <Link
+            href="/principles/themes/#opinionated-theme-inputs-list"
+            passHref
+          >
+            <DxcLink>Themes section</DxcLink>
+          </Link>{" "}
+          section. We strongly recommend using the{" "}
+          <Link href="/theme-generator/advanced-theme" passHref>
+            <DxcLink>advanced theme generator</DxcLink>
+          </Link>{" "}
+          to create the theme object that will be used in the HalstackProvider.
+        </DxcParagraph>
+        <Example example={advancedTheme} defaultIsVisible />
+
+        <DxcParagraph>
+          In the example below we have customized some of the DxcAccordion
+          tokens. As you can see, it is not necessary to pass all the tokens of
+          the component, only those that change their value with respect to the
+          default theme.
+        </DxcParagraph>
+      </>
+    ),
+  },
+  {
+    title: "Localization",
+    content: (
+      <>
+        <DxcParagraph>
+          HalstackProvider can be used to translate all the labels that cannot
+          be changed by the component properties.
+        </DxcParagraph>
+        <DxcParagraph>
+          To find out the list of labels that are translatable we should refer
+          to the documentation in the{" "}
+          <Link href="/principles/localization" passHref>
+            <DxcLink>Localization</DxcLink>
+          </Link>{" "}
+          section.
+        </DxcParagraph>
+        <DxcParagraph>
+          Let's imagine that we want to translate the '(Optional)' label of the
+          components that show it, as well as the error messages of our File
+          Input component (to see the translated error message you should try to
+          add any file on fileinput), we should create an object that will
```

Comment:
> Maybe after the parenthesis it better fits a point instead of a comma, the sentences are not connected at all.
> 
> > Let's imagine that we want to translate the '(Optional)' label of the components that show it, as well as the error messages of our File Input component. **To do this, we need to create an object very similar in structure to the themes, containing the desired components and their affected labels.**

Done.



> In regards to the explanation of how to see the translated error message, I suggest adding it in the code of the example with a `DxcParagraph`. It is more related to the example than to the actual explanation of localization, so it should not cloud the text.

I´ve changed the way the explanation is displayed. Please take a look.

Reply:
It is perfect! Just a quick note, instead of "...as <much> objects as components..." it would be more correct to say "...as many objects as components..."

Reply:
Changed.